.rustfmt.toml

```diff
@@ -0,0 +1,5 @@
+reorder_imports = true
+
+# These only work with `cargo +nightly fmt`
+# imports_granularity = "Module"
+# group_imports = "StdExternalCrate"
```

Comment:
For now I think it's ok and we can run it manually, but later we should probably make it enabled by default (in CI as well).
Honestly I'd like to add to the repo `.vscode` folder with `launch.json` and `settings.json` with enabled formatting on save (nightly) and some other predefined flags to make things more universal for everyone. But that's for later too.

Reply:
Enabling it by default means that we are forced to use `cargo +nightly fmt`. Are you OK with that? If so, I'm happy to leave them enabled.

When someone runs `cargo fmt` with these options enabled, they get some warnings like:

```
$ cargo fmt
Warning: can't set `imports_granularity = Module`, unstable features are only available in nightly channel.
Warning: can't set `group_imports = StdExternalCrate`, unstable features are only available in nightly channel.
Warning: can't set `imports_granularity = Module`, unstable features are only available in nightly channel.
Warning: can't set `group_imports = StdExternalCrate`, unstable features are only available in nightly channel.
Warning: can't set `imports_granularity = Module`, unstable features are only available in nightly channel.
Warning: can't set `group_imports = StdExternalCrate`, unstable features are only available in nightly channel.
```

Reply:
I'd say let's do it. We can also add recommendation for newcomers to install and use nighlty for the formatting.
And later configure things in **.vscode/settings.json**.